purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1554,6 +1554,10 @@ class Purchases internal constructor(
         presentedOfferingIdentifier: String?,
         listener: PurchaseErrorCallback
     ) {
+        // TODO why do we use findPurchaseInPurchaseHistory instead of queryPurchases?
```

Comment:
+1 Deema and I had the same question

Reply:
@vegaro do you have any historical context on this? if not, i'll go ahead and make the change.

Reply:
@vegaro bump

Reply:
Hmmm I am not sure, but the comment makes sense, although do we really have to make this change now? We can probably make the change later since it's an improvement that maybe changes something we are not sure about

Reply:
perfect is the enemy of shipped